tic-tac-toe-test/RulesTest.cs

```diff
@@ -0,0 +1,79 @@
+using Xunit;
+using tic_tac_toe;
+using System.Collections.Generic;
+
+namespace tic_tac_toe_test
+{
+    public class RulesTest
+    {
+        [Fact]
+        public void GameIsNotOverWhenThereIsNoWinner()
+        {
+            List<char> notFullBoardState = new List<char> {
+                '_', 'O', 'X',
+                'O', '_', 'O',
+                'X', 'O', '_',
+                };
+
+            Board board = new Board(notFullBoardState);
+
+            Assert.False(Rules.GameOver(board));
+        }
+
+        [Fact]
+        public void GameIsOverWhenBoardIsFull()
+        {
+            List<char> fullBoardState = new List<char> {
```

Comment:
This test might also assert true that the 'x' player also wins on the diagonal, not just that the board is full

Reply:
Good spot! I should change the board state to have no winner, then the test might be clearer.